test/unit/builtins/string.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { LuaLibImportKind } from "../../../src";
 import * as util from "../../util";
+import { describe } from "jest-circus";
```

Comment:
This should not be required, I don't think we import describe in any other file.

Reply:
I'm going to blame auto-import for this